### DISCUSSION TITLE
Weird errors when installing lazyvim in docker

### DISCUSSION THREAD

**Discussion by Bloodsnake** [LazyVim/LazyVim#2699] [Q&A]
Hello, I tried installing neovim and lazyvim inside a docker container. Here is the Dockerfile I wrote for this:
```Dockerfile
FROM debian:latest

# init apt
RUN apt update -y && apt upgrade -y && apt autoremove -y

# install base packages and nivm dependencies
RUN apt install -y \
    git \
    iputils-ping \
    curl \
    unzip \
    fish \
    # nvim
    ninja-build \
    gettext \
    cmake
    
# build and install neovim
RUN cd /tmp && git clone https://github.com/neovim/neovim && \
    cd neovim && \
    git checkout stable && \
    make CMAKE_BUILD_TYPE=Release && \
    make install && \
    cd .. && \
    rm -rf neovim && \
    apt remove ninja-build -y

# layzvim
RUN git clone https://github.com/LazyVim/starter ~/.config/nvim && \
    rm -rf ~/.config/nvim/.git && \
    nvim --headless +LazyHealth +q

# Avoid container exit.
CMD ["tail", "-f", "/dev/null"]
```
After building the image and running the container I go into the container by executing: `docker exec -it dev-container /bin/fish`
When I first run `nvim` and the plugins are supposed to get installed I get these errors after some seconds:
![Screenshot_20240311_014604](https://github.com/LazyVim/LazyVim/assets/58860946/79a34fa7-7f42-4a3a-b3da-85089963580d)
![Screenshot_20240311_014448](https://github.com/LazyVim/LazyVim/assets/58860946/7f0026b1-e7f7-4d4e-9d6c-d78b11f3d346)
I am able to to ping `github.com` inside the container.

**Comment by 0x2E** [comment]
No error in my container. Seems like your network router issue.
